service-api/src/main/java/greencity/dto/CourierTranslationDto.java

```diff
@@ -8,6 +8,9 @@
 @EqualsAndHashCode
 @ToString
 @Builder
+/**
+ * {@inheritDoc}
+ */
```

Comment:
Please use valid documentation instead of {@inheritDoc}. If I try to generate Java Docs I will not get anything.